src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -124,6 +127,86 @@ protected function json($data, $status = 200, $headers = array(), $context = arr
     }
 
     /**
+     * Returns a BinaryFileResponse object with original or customized file name and disposition header.
+     *
+     * @param File|string $file        File object, path to file or string with content to be sent as response
+     * @param null        $fileName    File name to be sent to response or null (will use original file name)
+     * @param string      $disposition Disposition of response (attachment is default, other type is inline)
+     *
+     * @return BinaryFileResponse
+     *
+     * @throws \LogicException|\InvalidArgumentException|\Exception
+     */
+    protected function file($file, $fileName = null, $disposition = ResponseHeaderBag::DISPOSITION_ATTACHMENT)
+    {
+        $deleteFileAfterSend = false;
+
+        // Test if path to file is given
+        if (is_string($file) && file_exists($file)) {
+            $file = new File($file);
+        }
+
+        // Test if content in string is given
+        if (is_string($file)) {
+            if (empty($file)) {
```

Comment:
Symfony style prefers strict comparison when possible, so it should be `'' === $file`